tests/test-mongoc-client-pool.c

```diff
@@ -38,9 +37,37 @@ test_mongoc_client_pool_try_pop (void)
 }
 
 
+static void
+test_mongoc_client_pool_dispose_old (void)
+{
+   mongoc_client_pool_t *pool;
+   mongoc_client_t *client;
+   mongoc_uri_t *uri;
+   int c;
+
+      uri = mongoc_uri_new("mongodb://127.0.0.1?maxpoolsize=3&minpoolsize=2");
+      pool = mongoc_client_pool_new(uri);
+
+      mongoc_client_t *clients[3];
```

Comment:
msvc unfortunately demands variable declarations at the top of the block, so this needs to move up by the other declarations